### DISCUSSION TITLE
Incorrect response.  Please refresh the page.

### DISCUSSION THREAD

**Discussion by ZelabDK** [vogler/free-games-claimer#266] [General]
Cannot log in to epic games, it throws the error seen below. Refreshing doesn't fix the error.

![Skærmbillede 2023-12-10 011559](https://github.com/vogler/free-games-claimer/assets/76261057/3b207979-450b-4c8e-b82c-ecd97f664712)

**Comment by IsT3RiK** [comment]
Hello same for me i check the captcha and they never want to go after.

![Screenshot_10](https://github.com/vogler/free-games-claimer/assets/91830375/31949d59-3abb-45cd-b95f-0ace1dc2b02d)
![Screenshot_11](https://github.com/vogler/free-games-claimer/assets/91830375/1875b511-e09d-45fa-a820-98d5709586d5)

**Reply by HolgerM1** [reply]
![Screenshot 2024-03-17 154503](https://github.com/vogler/free-games-claimer/assets/114694650/36c4e954-d9e7-419c-ae4d-8f3c3b7faffd)

same issue here...

**Comment by mellman99** [comment]
any resolution to this?

**Comment by marcel-v90** [comment]
I also have this error with Epic Games. Is there a solution?

**Comment by EnduringGuerila** [comment]
I have been trying on and off to get this script to work for almost a year. 
It worked briefly a few months ago and then just stopped after like 6 weeks. 
The main error I'm getting now when trying to set it back up is this ^
No matter what I try on the captcha, always fails.

**Comment by xamindar** [comment]
Yeah this is broke. This container has not worked for over a year for me, probably longer. I have it in a cron and get the timeout notification every day after it fails to log in to epic. When troubleshooting I get the same errors listed here. Two captcha come up and after solving them, it just goes back to password entry saying "Incorrect response. Please refresh the page."

I think it's just time to stop using this thing. I have been claiming the games manually for the past couple years anyway because this one fails to work.
